docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+
+The following command can be used to see the percentage completion of the sync only:
+
+    nix-shell -p jq curl --run 'curl -X GET "https://127.0.0.1:8090/api/v1/node-info" -H "accept: application/json;charset=utf-8" | jq .data.syncProgress.quantity'
+
+## Creating a New Wallet
+You can create your first wallet using the POST /api/v1/wallets endpoint as follow:
+
+    curl -X POST https://localhost:8090/api/v1/wallets                     \
+         -H "Content-Type: application/json; charset=utf-8"                \
+         -H "Accept: application/json; charset=utf-8"                      \
+         --cacert ./scripts/tls-files/ca.crt                               \
+         -d '{                                                             \
+      "operation": "create",                                               \
+      "backupPhrase": ["squirrel", "material", "silly", "twice", "direct", \
+        "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],  \
+      "assuranceLevel": "normal",                                          \
+      "name": "MyFirstWallet"                                              \
+    }'
+Warning: Those 12 mnemonic words given for the backup phrase act as an example. Do not use them on a production system. See the section below about mnemonic codes for more information.
+
+As a response, the API provides you with a wallet id used in subsequent requests to uniquely identity the wallet. Make sure to store it / write it down. Note that every API response is jsend-compliant; Cardano also augments responses with meta-data specific to pagination. More details in the section below about Pagination.
+
+    {
+        "status": "success",
+        "data": {
+            "id": "Ae2tdPwUPE...8V3AVTnqGZ",
+            "name": "MyFirstWallet",
+            "balance": 0
+        },
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 1,
+                "totalEntries": 1
+            }
+        }
+    }
+You have just created your first wallet. Information about this wallet can be retrieved using the GET /api/v1/wallets/{walletId} endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}} \
+         -H "Accept: application/json; charset=utf-8"              \
+         --cacert ./scripts/tls-files/ca.crt                       \
+
+## Receiving Money
+
+To receive money from other users you should provide your address. This address can be obtained from an account. Each wallet contains at least one account, you can think of account as a pocket inside of your wallet. Besides, you can view all existing accounts of a wallet by using the GET /api/v1/wallets/{{walletId}}/accounts endpoint as follow:
```

Comment:
s/Besides, you can v/V/

Reply:
Remove "as follow"

Reply:
> Each wallet contains at least one account, you can think of account as a pocket inside of your wallet.

How about "Each wallet contains at least one account. An account is like a pocket inside your wallet."